### ISSUE TITLE
ONNX - Convert

### ISSUE THREAD

**Issue by junxnone** [junxnone/tio#739]
# Reference
- [ ] [pytorch_to_onnx.py - openvino_training_extensions](https://github.com/opencv/openvino_training_extensions/blob/develop/pytorch_toolkit/utils/pytorch_to_onnx.py)
- [ ] [mask r-cnn pytorch2onnx  - openvino_training_extensions](https://github.com/opencv/openvino_training_extensions/blob/develop/pytorch_toolkit/instance_segmentation/tools/convert_to_onnx.py)
- [ ] [pretrained alexnet - pytorch2onnx](https://colab.research.google.com/drive/1QVbxv12YKPIoSKNOoHpNxKHdYofEgrgm)

# Torch
```
torch.onnx.export(model, args, f, export_params=True, verbose=False, training=False, \
input_names=None, output_names=None, aten=False, export_raw_ir=False, \
operator_export_type=None, opset_version=None, _retain_param_name=True, \
do_constant_folding=True, example_outputs=None, strip_doc_string=True, dynamic_axes=None, \
keep_initializers_as_inputs=None, custom_opsets=None, enable_onnx_checker=True, \
use_external_data_format=False)
```
- export alexnet
```
torch.onnx.export(model, dummy_input, "alexnet.onnx", verbose=True,\
 input_names=input_names, output_names=output_names)
```

**Comment by junxnone** [comment]
#734
